ml.go

```diff
@@ -13,8 +14,7 @@ type memberList struct {
 	uncontacted map[id]bool // ids that have not been sent to
 	removed     map[id]bool // removed ids // TODO: expire old entries by timestamp
 
-	order []id
-	i     int
+	o order.Order[id]
```

Comment:
Maybe the package `order` could have a more descriptive name. It could say what kind of order it is. Like `roundrobin` or `random`? Personally I like the idea of calling it `roundrobinrandom` :)

Reply:
> `roundrobinrandom`

That _is_ what the package does, after all. It's 16 characters, which I guess isn't too many for an internal package...